docs/Milestones.md

```diff
@@ -0,0 +1,8 @@
+# Milestones
+> Hitos
+
+## [[M0] Estructura de almacenamiento de canciones](https://github.com/Carlos-SE/App-Chores/milestone/1)
+Los usuarios necesitan canciones en base a sus gustos musicales. Es por esto que se necesita desarrollar una estructura que modelice las canciones. Aalmacenando tanto los datos de la canción, como el archivo de la canción en cuestión.
+
+## [[M1] Algoritmo de recomendación en base a gustos](https://github.com/Carlos-SE/App-Chores/milestone/2)
+Los usuarios necesitan un algoritmo que les recomiende canciones nuevas y que estén adaptadas a sus gustos. Ya sea por géneros, artistas, etc.
```

Comment:
Un algoritmo no es un produco.

Reply:
@JJ @pablo1mc315 si podéis revisar el PR de nuevo os lo agradecería ^-^

Reply:
"Recomendador" tampoco. Y pone canciones

Reply:
:arrow_up: